modules/distr-tracing-tempo-config-query-frontend.adoc

```diff
@@ -6,62 +6,125 @@
 [id="distr-tracing-tempo-config-query_{context}"]
 = Query configuration options
 
-Query is a service that retrieves traces from storage and hosts the user interface to display them.
+Queries are managed in the {TempoShortName} by using two of its components, the querier and query frontend, both of which are configurable.
```

Comment:
This would be better, in my view, if you use the active voice and break the sentence down. Something like this:

Two components of {TempoShortName}, the querier and query frontend, manage the queries. You can configure both of these components.

Reply:
Great suggestion. Thank you. Applying.